Source/DataSources/EntityCluster.js

```diff
@@ -659,7 +659,10 @@ function createGetEntity(
 
     entityIndices[entityIndexProperty] = index;
 
-    this._clusterDirty = true;
+    const that = this;
+    Promise.resolve(entityItem.ready).then(function () {
```

Comment:
Is this code intentional? EntityCluster as a whole is confusingly implemented, but I tried to figure out what entityItem was and it seems to be different depending on the item (like a billboard or a label)  and while sometimes `entityItem.ready` will be something that exists, it's never a promise, only a boolean. So why is the Promise.resolve here?